deepreg/model/loss/image.py

```diff
@@ -3,44 +3,119 @@
 """
 import tensorflow as tf
 
+from deepreg.model.loss.util import NegativeLossMixin
+from deepreg.registry import REGISTRY
+
 EPS = tf.keras.backend.epsilon()
 
 
-def dissimilarity_fn(
-    y_true: tf.Tensor, y_pred: tf.Tensor, name: str, **kwargs
-) -> tf.Tensor:
-    """
-    Returns the calculated dissimilarity for each batch.
+@REGISTRY.register_loss(name="ssd")
+class SumSquaredDistance(tf.keras.losses.Loss):
```

Comment:
it is better to be SumSquaredDifferences or  SumSquaredDiff? - distance can be confusing

Reply:
Ok

Reply:
done